### PR TITLE
Fix register existing emails redirect and add redirects to additional…

### PR DESCRIPTION
**Pull request by leomendoza123**
… emails

### CODE REVIEW

**Comment by leomendoza123** [comment]
[When an email entered on the Registration screen is already in the db, auto-populate it when the user clicks to sign in](https://trello.com/c/Rmi0WGbx/6854-when-an-email-entered-on-the-registration-screen-is-already-in-the-db-auto-populate-it-when-the-user-clicks-to-sign-in)

File: src/app/register/components/form-personal-additional-emails/form-personal-additional-emails.component.html [hunk 1]
```diff
@@ -37,7 +37,10 @@
           )
         "
       >
-        <app-backend-error [errorCode]="error"></app-backend-error>
+        <app-backend-error
+          [errorCode]="error"
+          [value]="additionalEmailControl.value.value"
+        ></app-backend-error>
```
Comment (DanielPalafox) [inline src/app/register/components/form-personal-additional-emails/form-personal-additional-emails.component.html:43]: I think `value` it is duplicated here

Comment (leomendoza123) [reply src/app/register/components/form-personal-additional-emails/form-personal-additional-emails.component.html:43]: is weird, but is the way it is

**Comment by leomendoza123** [comment]
[[QA] register - fix error message for emails with deactivated and unclaimed accounts](https://trello.com/c/nivDnc2z/6908-qa-register-fix-error-message-for-emails-with-deactivated-and-unclaimed-accounts)

**Pull request merged**